File: NutiteqSdkExample/app/src/main/java/com/example/rohit/nutiteqsdkexample/MainActivity.java
package com.example.rohit.nutiteqsdkexample;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.nutiteq.ui.MapView;

public class MainActivity extends AppCompatActivity
{
    private MapView mapView;



    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        MapView.registerLicense("XTUN3Q0ZFRWxWSEtCSUk3bVJxN0pQYnRxc3dFb0ovaGhBaFE5TUN3MWFwZTVTRnptSm1oam5VV3RpaWVwL1E9PQoKcHJvZHVjdHM9c2RrLWFuZHJvaWQtMy4qCnBhY2thZ2VOYW1lPWNvbS5leGFtcGxlLnJvaGl0Lm51dGl0ZXFzZGtleGFtcGxlCndhdGVybWFyaz1udXRpdGVxCm9ubGluZUxpY2Vuc2U9MQp1c2VyS2V5PTQzZWU2MmJkNjk2NDNhNGQzMmNkMzUzMjk4NTdmM2MwCg==",this);
       // mapView = (MapView)this.findViewById(R.id.)
    }



}

File: CalculatorApp/app/src/main/java/com/example/rohit/calculatorapp/MainActivity.kt
package com.example.rohit.calculatorapp

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast
import com.example.rohit.calculatorapp.R.id.addi
import kotlinx.android.synthetic.main.activity_main.*
import java.util.*

class MainActivity : AppCompatActivity()
{

    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)




        addi.setOnClickListener{
            Toast.makeText(this@MainActivity, "You clicked me.", Toast.LENGTH_SHORT).show()
            var firstno  = first_no.text.toString()
            var secondno = second_no.text.toString()
            var add = firstno.toInt() + secondno.toInt();
            result.text = add.toString()

        }
        sub.setOnClickListener{
            Toast.makeText(this@MainActivity, "You clicked me.", Toast.LENGTH_SHORT).show()
            var firstno  = first_no.text.toString()
            var secondno = second_no.text.toString()
            var add = firstno.toInt() - secondno.toInt();
            result.text = add.toString()

        }
        mult.setOnClickListener{
            Toast.makeText(this@MainActivity, "You clicked me.", Toast.LENGTH_SHORT).show()
            var firstno  = first_no.text.toString()
            var secondno = second_no.text.toString()
            var add = firstno.toInt() * secondno.toInt();
            result.text = add.toString()
        }
        div.setOnClickListener{
            Toast.makeText(this@MainActivity, "You clicked me.", Toast.LENGTH_SHORT).show()
            var firstno  = first_no.text.toString()
            var secondno = second_no.text.toString()
            var add = firstno.toInt() / secondno.toInt();
            result.text = add.toString()
        }



    }
}
